source/MainPanel.cpp

```diff
@@ -151,16 +156,18 @@ void MainPanel::Draw()
 
 	if(Preferences::Has("Show CPU / GPU load"))
 	{
-		string loadString = to_string(lround(load * 100.)) + "% GPU";
+		auto &font = FontSet::Get(14);
 		const Color &color = *GameData::Colors().Get("medium");
-		FontSet::Get(14).Draw(loadString, Point(10., Screen::Height() * -.5 + 5.), color);
 
-		loadSum += loadTimer.Time();
-		if(++loadCount == 60)
 		{
-			load = loadSum;
-			loadSum = 0.;
-			loadCount = 0;
+			float sec = loadTimer.Time();
+			string loadString = to_string(lround(sec * 1000.)) + "ms GPU";
+			font.Draw(loadString, Point(10., Screen::Height() * -.5 + 5.), color);
+		}
```

Comment:
What are these additional scopes for?

Reply:
preventing name collisions, I'll change them

Reply:
You can just assign a new value to the old variable, no?